Amplify/Core/Support/Cancellable.swift

```diff
@@ -10,3 +10,5 @@
 public protocol Cancellable {
     func cancel()
 }
+
+public typealias AmplifyCancellable = Cancellable
```

Comment:
Can you add a brief note explaining why we have this, and that it may be removable once we drop support for iOS < 13?

Reply:
I've added a comment to provide context for this addition.